BaseFilter/sections/antiadblock.txt

```diff
@@ -1213,6 +1213,12 @@ youporn.com$$script:contains(/window\.[\s\S]*?_zone_[\s\S]*?{"zone_id":/)
 !
 ! NOTE: Regular rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/221388
+goonchan.org#%#//scriptlet('prevent-setTimeout', 'adsbygoogle')
+goonchan.org#%#//scriptlet('abort-on-stack-trace', 'open', 'onClick')
+goonchan.org#@#ins.adsbygoogle[data-ad-slot]
+goonchan.org#@#ins.adsbygoogle[data-ad-client]
+goonchan.org##iframe[src^="https://creative.mavrtracktor.com/"]

```

Comment:
Please move:
```adblock
goonchan.org##iframe[src^="https://creative.mavrtracktor.com/"]
goonchan.org#%#//scriptlet('abort-on-stack-trace', 'open', 'onClick')
```
to `Specific rules` section in `https://github.com/AdguardTeam/AdguardFilters/blob/master/BaseFilter/sections/specific.txt`, as it's not related to anti-adblock.